scripts/test.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if ! [ -a .build-test ] ; then
+	echo "Creating Virtual Environment"
+	virtualenv .build-test -p python3
+fi
+source .build-test/bin/activate
+pip3 -q install -r requirements_for_test.txt
+pytest --cov=swagger_server/controllers_local --cov-report term-missing
```

Comment:
Mmm, they don't do the same thing. run_unit_test.sh is there for anyone working with 'workon' who wants html versions of reports, test.sh is there for virtenv users who just want cli information. Yes you could put the code in the same file and parameterise, but it's not something that would be done in codegen as these are auto-generated.